Fill.py

```diff
@@ -578,7 +597,7 @@ def failed(warning: str, force):
             elif target_world is None:  # target all worlds
                 worlds = set(world.player_ids)
             elif type(target_world) == list:  # list of target worlds
-                worlds = []
+                worlds = []  # TODO: why do we make list and convert to set, instead of just set to begin with?
```

Comment:
If there are copies of the world appearing in the list, converting to set would remove them. Having a set in the first place would throw an exception in the case of a copy being added.

I'm guessing here based on what I know about @Berserker66's coding habits, but I don't even know if he wrote this.

Reply:
`add` doesn't throw an exception if a copy is added

```python3
>>> s = set()
>>> s.add(2)
>>> s.add(2)
>>> 
```

Reply:
yes, set would be more efficient here